src/main/java/mate/academy/model/User.java

```diff
@@ -53,8 +57,25 @@ public void setSalt(byte[] salt) {
     @Override
     public String toString() {
```

Comment:
Sensitive fields like `password` and `salt` should not be included in the `toString` method to prevent accidental exposure of sensitive information.